src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -971,6 +1016,32 @@ public float CalculateTotalDigestionEfficiency()
         return MicrobeInternalCalculations.CalculateTotalDigestionEfficiency(editedMicrobeOrganelles);
     }
 
+    public override void OnLightLevelChanged(float lightLevel)
+    {
+        var maxLightLevel = Editor.CurrentPatch.GetCompoundAmount("sunlight", CompoundAmountType.Maximum);
+        var templateMaxLightLevel = Editor.CurrentPatch.GetCompoundAmount("sunlight", CompoundAmountType.Template);
+
+        if (maxLightLevel > 0.0f && templateMaxLightLevel > 0.0f)
```

Comment:
Shouldn't either of these being more than 0 work? We might want to allow patches to deviate from the template in terms of max light level.

Reply:
The issue here is that only some backgrounds have been given proper nighttime colouring, specifically those whose templates have non-zero light level. The sea floor for instance has zero sunlight in its definition but can be given non-zero sunlight as a patch instance. @Nunez2196 must have missed this and didn't provide a proper background shader, so it just turns pitch black at night without this check.

I think this should be raised as an issue to deal with in future. For now I'll definitely add a comment here.